admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,550 @@
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[Managing Role Bindings].
+
+=== Adding a Role to a User
+
+To bind a role to a user for the current project:
+
+----
+$ oadm policy add-role-to-user <role> <user_name>
+----
+
+You can specify a project with the `-n` flag.
+
+=== Removing a Role from a User
+
+To remove a role from a user for the current project:
+
+----
+$ oadm policy remove-role-from-user <role> <user_name>
+----
+
+You can specify a project with the `-n` flag.
+
+=== Adding a Role to a User for All Projects
+
+To bind a role to a user for all projects in the cluster:
+
+----
+$ oadm policy add-cluster-role-to-user <role> <user_name>
+----
+
+=== Removing a Role from a User for All Projects
+
+To remove a role from a user for all projects in the cluster:
+
+----
+$ oadm policy remove-cluster-role-from-user <role> <user_name>
+----
+
+=== Adding a Role to a Group
+
+To bind a role to a specified group in the current project:
+
+----
+$ oadm policy add-role-to-group <role> <groupname>
+----
+
+You can specify a project with the `-n` flag.
+
+=== Removing a Role from a Group
+
+To remove a role from a specified group in the current project:
+
+----
+$ oadm policy remove-role-from-group <role> <groupname>
+----
+
+You can specify a project with the `-n` flag.
+
+=== Adding a Role to a Group for All Projects
+
+To bind a role to a specified group for all projects in the cluster:
+
+----
+$ oadm policy add-cluster-role-to-group <role> <groupname>
+----
+
+=== Removing a Role from a Group for All Projects
+
+To remove a role from a specified group for all projects in the cluster:
+
+----
+$ oadm policy remove-cluster-role-from-group <role> <groupname>
+----
+
+[[share-templates-cluster]]
+== Sharing Templates for Use in Projects Across the Cluster
+
+Templates are project-scoped resources, so you cannot create them to be readily
+available at a cluster level. The easiest way to share templates across the entire
+cluster is with the `openshift` project, which by default is already set up to
+share templates. The templates can be annotated, and are displayed in the web
+console where users can access them. Users have `get` access only to the
+templates and images in this project, via the `shared-resource-viewer` role.
+
+The `shared-resource-viewer` role exists to allow templates to be shared across
+project boundaries. Granting a user this role for a project gives them the
+ability to see all existing templates and pull images from that project.
+However, the user still needs to know which project to look in, because they
+will not be able to view the project in their `oc get projects` list.
+
+If you would prefer to share templates across the cluster using your own
+project instead of the default `openshift` project, you can grant users the
+`shared-resource-viewer` role with that project:
+
+----
+$ oadm policy add-role-to-user shared-resource-viewer <user_name> -n <project_name>
```

Comment:
Because of the way we built it, this won't work.  The role only exists in the openshift namespace.  A cluster-admin can use `oc export role/shared-resource-viewer -o yaml -n openshift | oc create -n new-namespace -f - && oadm policy add-role-to-user shared-resource-viewer <user_name> -n new-namespace --role-namespace=new-namespace`.

If you think its common, we can move it.